### DISCUSSION TITLE
Does Amazon Ads API support for Amazon.in?

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#7] [Q&A] [answered]
@elizafarley 

**Amazon.in** is not mentioned in docs. Is **amazon.in** not supported?

https://advertising.amazon.com/API/docs/en-us/info/api-overview#api-endpoints

<img width="809" alt="Screenshot 2022-07-01 at 3 25 42 PM" src="https://user-images.githubusercontent.com/28811774/176872316-456ceb4b-a15f-43ed-ba98-81ffa05fdad4.png">

**Comment by elizafarley** [comment, marked as answer]
The Ads API currently supports Sponsored Display endpoints in India. To access those endpoints for the India marketplace, you should use the EU URL `https://advertising-api-eu.amazon.com`. 

To see what SD endpoints are available in the IN marketplace, take a look at our [Sponsored Display features by marketplace doc](https://advertising.amazon.com/API/docs/en-us/sponsored-display/features).

**Comment by rajatanand86** [comment]
Hello can someone point how to access this api.
going to below shows united states as country and when india is selected, no register form is displayed.
https://advertising.amazon.com/partner-network/register-api?ref_=a20m_us_api_drctad
